icu4c/source/test/intltest/measfmttest.cpp

```diff
@@ -82,6 +82,7 @@ class MeasureFormatTest : public IntlTest {
     void TestInvalidIdentifiers();
     void TestCompoundUnitOperations();
     void TestIdentifiers();
+    void TestAddressSanitizerProblem();
```

Comment:
Suggested:

```cpp
void Test21060_AddressSanitizerProblem();
```

in case there's another address sanitizer problem in the future.

Reply:
Done (i.e. coming up in my next push). I've also moved the actual body of this code to be below the body of TestIdentifiers, FWIW. In the body, I improved the comments.